core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -353,7 +356,7 @@ public Handler configureWebSocketHandler() throws ServletException {
     configureSecurityHandler(webSocketContext);
 
     ServerContainer container =
-            WebSocketServerContainerInitializer.configureContext(webSocketContext);
+            WebSocketServerContainerInitializer.initialize(webSocketContext);
```

Comment:
Good point. The behavior is correct with this change, but future changes in the internal API would break use. The new API is different and will affect other repositories. So, we have two options:

1. Continue using the deprecated API with warning suppression
2. Use the internal API

Either way, I'll file a JIRA to investigate using the new non deprecated API.

Reply:
@dimitarndimitrov Between `1` and `2`, do you have a preference? If you have an alternative suggestion, that's welcome too.

Reply:
I think we should go with `2` until the JIRA for the new non-deprecated API gets resolved - `1` just directly calls the API from `2`, but in addition it also generates a warning, can break us if the JIRA for the new API stalls, etc.

Reply:
In that case, I think this is ready for another round of reviews. I'd like to get this merged soon as there is an approved PR in AK for removing the `Total` class.